packages/fxa-js-client/client/FxAccountClient.js

```diff
@@ -2471,6 +2471,75 @@ define([
       });
   };
 
+  /**
+   * Get a user's list of active subscriptions.
+   *
+   * @param {String} authorizationHeader
+   * The authorization header for the request.
+   * It should be in the form of `Bearer ${SOME_TOKEN}`.
```

Comment:
I think it would be a nicer API if the `Bearer ${refreshToken}` concatenation happened inside this lib. Then callers would only have to worry about passing `refreshToken` into these methods, so less work for them and it's also consistent with how the other token types are passed in.

Reply:
Made the suggested change.